packages/xdl/src/detach/IosPlist.ts

```diff
@@ -93,18 +93,4 @@ async function createBlankAsync(plistPath: string, plistName: string) {
   fs.removeSync(tmpConfigFile);
 }
 
-async function cleanBackupAsync(plistPath: string, plistName: string, restoreOriginal = true) {
```

Comment:
No longer called anywhere.

This entire file seems like it should be phased out eventually.

Reply:
most of this xdl stuff was supposed to be legacy turtle v1 things that we replaced with config plugins, but they're still referenced a bit in export/publish